cli/inspect.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
```

Comment:
Please take a look of https://github.com/alibaba/pouch/blob/master/docs/contributions/code_styles.md#additional-style-rules, the forth point.